File: PANA/PANA2020/C.cpp
#include <boost/multiprecision/cpp_dec_float.hpp>
#include <boost/multiprecision/cpp_int.hpp>
#include <iostream>

int main() {
  using namespace std;
  namespace mp = boost::multiprecision;
  using Real = mp::number<mp::cpp_dec_float<1024>>;

  Real a, b, c;
  cin >> a >> b >> c;

  if (mp::sqrt(a) + mp::sqrt(b) < mp::sqrt(c)) {
    cout << "Yes" << endl;
  } else {
    cout << "No" <<endl;
  }
}

File: PANA/PANA2020/C.py
from decimal import *

a, b, c = [int(x) for x in input().split(" ")]

A = Decimal(a)
B = Decimal(b)
C = Decimal(c)

if A.sqrt() + B.sqrt() < C.sqrt():
    print("Yes")
else:
    print("No")
